docs/docs/04-standard-library/µ-winglang_support_metrix.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Compatibility matrix
```

Comment:
why the file name change?

Reply:
it was the only capitalized one on he list, should I revert?